### PR TITLE
make cache health check aware of all cache backends

### PR DESCRIPTION
**Pull request by syphar**
First draft of what we see as a gap in functionality in this library. We do still use it heavily for our services. 

Here we didn't see a problem with one of the non-default cache backends quickly enough. 

I'm happy to change / improve where you deem it necessary. 

We do have this code running in production, so there is no time pressure, let's make it right :)

### CODE REVIEW

File: health_check/cache/backends.py
Comment (codingjoe) [inline health_check/cache/backends.py]: I believe it's more verbose to have the default value in the signature.

```suggestion
    def __init__(self, backend='default'):
        super().__init__()
        self.backend = backend
```

File: health_check/cache/backends.py
Comment (codingjoe) [inline health_check/cache/backends.py]: Python version support is 3.6 and up, feel free to use f-strings.
```suggestion
        return f"Cache backend: {self.backend}"
```

**Review by codingjoe — COMMENTED** [review]
Hi @syphar, excellent point. I agree that it does make sense to test all configured cache backends by default. I have merely one small suggestions. Maybe have a look. I'll merge this, once you had time to review my suggestions. Best, Joe

**Comment by syphar** [comment]
@codingjoe thanks! 

added both of your suggestions

**Review by codingjoe — APPROVED** [review]

**Review by codingjoe — APPROVED** [review]
👌

**Comment by syphar** [comment]
thanks @codingjoe :) 

Then we'll wait for the next release so we can remove the workaround

**Comment by codingjoe** [comment]
Released in [3.14.0](https://github.com/KristianOellegaard/django-health-check/releases/tag/3.14.0) 🎉 

Auto-releases on PyPi are currently broken, but you get special treatment. I released it manually ;)

**Pull request merged**
